.github/workflows/pre-commit.yml

```diff
@@ -24,6 +24,8 @@ jobs:
           node-version-file: .nvmrc
           cache: pnpm
       - run: pnpm --color install
+      - uses: leafo/gh-actions-lua@v9
+      - uses: leafo/gh-actions-luarocks@v4
```

Comment:
Do you need these? Usually pre-commit steps will create their own isolated environment with everything they need

Reply:
Without these, pre commit fails due to missing pistols binary https://github.com/cursorless-dev/cursorless/actions/runs/9430140303/job/25977330727?pr=2256
So afaict these are needed? Or am I missing something?

Reply:
I _think_ that might've been one of the reasons I had used the non official pre-commit repos is that they were setup to install the luarocks, etc requirements, whereas the official ones didn't.

Reply:
Yes, it was pretty annoying to install luarocks and dependencies on Windows locally tbh lol. Required gcc etc. But I did manage to get it working